.github/workflows/test-harness-verity-afj.yml

```diff
@@ -0,0 +1,53 @@
+name: test-harness-verity-afj
+# RUNSET_NAME: "Verity to Javascript"
+# Scope: AIP 1.0
+# Exceptions: None
```

Comment:
Please update to be accurate. This appears in the summary table of the Info page.  For example, are we doing anything other than connection?

Also, please add an "# Skip" line below this so that these results are not included in the Interop Summary until we are ready.